test/runtests.jl

```diff
@@ -4,5 +4,5 @@ using Test, TestSetExtensions
 using WrappedMultiGraphs
 
 @testset "AttributeGraphs.jl" begin
-    @includetests ["constructors","graphs", "attributes"]
+    @includetests ["constructors","graphs", "attributes",  "opinionated"]
```

Comment:
just the spacing a bit
```suggestion
    @includetests ["constructors", "graphs",  "attributes",  "opinionated"]
```